ods-ci.yaml

```diff
@@ -0,0 +1,49 @@
+# PVC for storing the test-output artifacts created during a test run
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: ods-ci
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    deployment: ods-ci
+  name: ods-ci
+spec:
+  securityContext:
+    # This is required to make the test-output volume Writable
+    fsGroup: 0
+  containers:
+    - image: quay.io/spryor/ods-ci:latest
```

Comment:
I believe in QE we are starting to mantain an "official" image for ods-ci. @Xaenalt I don't know if this is a slimmed down version or not, but perhaps we could reuse the same. @bdattoma would have all the info on that.

Reply:
Yeah we can, that was just due to the official image at the time being very outdated and not running successfully

Reply:
you can use an image tag from https://quay.io/repository/modh/ods-ci. I'll provide more info offline